app/config/config.yml

```diff
@@ -76,6 +76,9 @@ doctrine:
             default:
                 connection: default
                 auto_mapping: false
+                metadata_cache_driver: array
+                query_cache_driver: array
+                result_cache_driver: array
```

Comment:
Afaik these values are already defined as `array` [by default](https://github.com/doctrine/DoctrineBundle/blob/master/DependencyInjection/Configuration.php#L617) in the DoctrineBundle, imo we should follow the default behaviour as much as possible. :)